src/instructlab/model/evaluate.py

```diff
@@ -101,6 +106,7 @@ def get_evaluator(
             branch,
             output_dir,
             max_workers,
+            merge_system_user_message=merge_system_user_message,
```

Comment:
This return is the case for mt_bench_branch right? I'm curious why you didn't use an else if or put a comment to signify that the final return case is for `mt_bench_branch`.

Reply:
I don't that was me who put in the if return logic previously.  It works fine but my preference would be to have an elif as well.